app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/AddOnsContainer.kt

```diff
@@ -0,0 +1,211 @@
+package com.kickstarter.ui.activities.compose.projectpage
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.kickstarter.ui.compose.designsystem.KSCoralBadge
+import com.kickstarter.ui.compose.designsystem.KSDividerLineGrey
+import com.kickstarter.ui.compose.designsystem.KSPrimaryBlackButton
+import com.kickstarter.ui.compose.designsystem.KSStepper
+import com.kickstarter.ui.compose.designsystem.KSTheme
+import com.kickstarter.ui.compose.designsystem.KSTheme.colors
+import com.kickstarter.ui.compose.designsystem.KSTheme.dimensions
+import com.kickstarter.ui.compose.designsystem.KSTheme.typography
+import com.kickstarter.ui.compose.designsystem.shapes
+
+@Composable
+@Preview(name = "Light", uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "Dark", uiMode = Configuration.UI_MODE_NIGHT_YES)
+private fun AddOnsContainerPreview() {
+    KSTheme {
+        AddOnsContainer(
```

Comment:
Not for now, if you guys are in a rush to deliver for project late pledges, but ideally we should start creating re-usable components for these type of cards.
The addOns one could potentially be re-used for rewards in both rewards-carousel and manage pledge screen.

Same for project cards in the future! 

cc: @ycheng-kickstarter @leighdouglas

Reply:
The functionality of the flows are so similar my eventual plan would be to use the existing screens with whatever copy change we need.  iOS is going to be using these in their existing flows.  If we had more time I would just refactor all the flows into these screens, but that will be a down the road initiative for sure.